man/builtin.doc

```diff
@@ -11138,12 +11182,6 @@ This performs as halt/1 above except that:
     \item abort() is called instead of exit(Status).
     \end{itemize}
 
-In addition to an integer status name we also allow passing a
```

Comment:
I guess you didn't want to remove my additions :smile:

Reply:
I blame git for removing your changes.